gke-policies/policy/nap_forbid_single_zone.rego

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: Ensure that node pool locations within Node Auto-Provisioning are covering more than one zone (or not enforced at all)
+# description: Node Auto-Provisioning configuration should cover more than one zone
+# custom:
+#   group: Security
+#   severity: Critical
+#   sccCategory: NAP_MULTIPLE_ZONES_CONFIGURED
```

Comment:
Please treat SCC category as something that describes violation.
Maybe sth like "NAP_ZONAL" like in `node_pool_multi_zone` policy?